FreeRTOS-Plus/Demo/AWS/Fleet_Provisioning_Windows_Simulator/Fleet_Provisioning_With_CSR_Demo/fleet_provisioning_serializer.c

```diff
@@ -0,0 +1,377 @@
+/*
```

Comment:
This file could be named to more clearly reference tinycbor and cbor in general.

Reply:
Resolved in this commit: https://github.com/FreeRTOS/FreeRTOS/pull/724/commits/9f65a80edcd6b5857abcfd0fd35bd576bd3cfd35

Reply:
Adjusted again in this commit: https://github.com/FreeRTOS/FreeRTOS/pull/724/commits/f1c5cdaa16b75521c57bfec65ed4c759bea63eb6